server/src/main/java/org/elasticsearch/search/profile/SearchProfileQueryPhaseResult.java

```diff
@@ -25,6 +26,8 @@
  */
 public class SearchProfileQueryPhaseResult implements Writeable {
 
+    private ProfileResult dfsProfileResult;
```

Comment:
If we wanted to measure the `termStatistics` calls, for example, we'd replace this with a `DfsProfileShardResult`, right?